### PR TITLE
docs: Added ReactJS Bangalore to React

### PR DESCRIPTION
**Pull request by saqlainkaleem**
## Adding a meetup
- [x] Suggestion is not a duplicate
- [x] Suggestion is under the proper category
- [x] Link to meetup is included 
- [x] Name of meetup is included
- [x] Name and link are spelled correctly

## Adding a category
- [x] Category is not a duplicate
- [x] Category is spelled correctly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,4 @@ We would love to know which meetups you think deserve to be included in our list
 - [React.js Girls](https://www.meetup.com/ReactJS-Girls-London/) - React.js Girls aims to voice women engineers who want to learn and teach their experience using React. All are welcomed!
 - [Reactjsday](https://2022.reactjsday.it/) - reactjsday is the Italian React conference, organized by GrUSP. The event’s goal is to allow all React developers to meet and share experience while having fun and networking in an enjoyable context.
 - [State of React](https://www.reactjsmeetup.com/#/) - State of React is a React meetup hosted multiple times a year by This Dot Media. Join a panel of React experts, enthusiasts, and contributors and get the opportunity to ask questions live!
+- [ReactJS Bangalore](https://www.meetup.com/ReactJS-Bangalore/) - ReactJS Bangalore organize meetups and workshops regularly to share knowledge and experiences about react ecosystem.
```
Comment (jdwilkin4) [inline README.md:38]:
```suggestion
- [ReactJS Bangalore](https://www.meetup.com/ReactJS-Bangalore/) - ReactJS Bangalore organizes meetups and workshops regularly to share knowledge and experiences about the React ecosystem.
```

Comment (jdwilkin4) [inline README.md:38]: This entry is out of order.
Please place it just after React Alicante

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: README.md (hunk 1 above)
Comment (saqlainkaleem) [reply README.md]: Hey All the required changes are done.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**